frb_codegen/src/source_graph.rs

```diff
@@ -158,10 +166,41 @@ impl Debug for Enum {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct ImplTrait {
+    pub ident: Ident,
+    pub src: TypeImplTrait,
+    pub bound: TraitBound,
+}
+#[derive(Debug, Clone, Eq)]
+pub struct Impl {
+    pub self_ty: Ident,
+    pub trait_: Ident,
+}
+
+impl Ord for Impl {
+    fn cmp(&self, other: &Impl) -> Ordering {
+        self.self_ty.cmp(&other.self_ty)
+    }
+}
+
+impl PartialOrd for Impl {
+    fn partial_cmp(&self, other: &Impl) -> Option<Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+impl PartialEq for Impl {
+    fn eq(&self, other: &Impl) -> bool {
+        self.self_ty == other.self_ty
+    }
+}
```

Comment:
I didn't find that, but here,
I just want to sort self_ty. It might be quicker to write it myself.
I would use it automatically, if you still suggest it.
Give me a reply.

> collect_impls_to_vec ' comment is not correct, I correct in my local

Reply:
such handwritten things are error-prone and hard to maintain, thus IMHO we should use the auto generated thing